cmd/ethrex/cli.rs

```diff
@@ -316,10 +318,10 @@ pub fn remove_db(datadir: &str, force: bool) {
     }
 }
 
-pub async fn import_blocks(path: &str, data_dir: &str, network: &str, evm: EvmEngine) {
+pub async fn import_blocks(path: &str, data_dir: &str, network: Networks, evm: EvmEngine) {
```

Comment:
review if all these functions need to receive a Network or a genesis_path: Path, or a Genesis